036-BGP/Coach/README.md

```diff
@@ -0,0 +1,57 @@
+

```

Comment:
This Coach table of contents README needs a bit of an overhaul... look at 001/Coach/README.md for a real world example.

Reply:
Coach guide ToC section is fine.  However, the "Deploying the Environment" & "Before You start" sections probably belong in either the solution for "Challenge 1" IMO.

Reply:
I have moved those two sections to challenge 1. Thanks!